sdk/communication/Azure.Communication.NetworkTraversal/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Release History
+
+## 1.0.0-beta.1 (2021-05-25)
+
+### Added
+- Added CommunicationRelayClient in preview.
+- Added CommunicationRelayClient.GetRelayConfiguration in preview.
+- Added CommunicationRelayClient.GetRelayConfigurationAsync in preview.
+
```

Comment:
In other packages like Chat and Common, we had this for initial public preview release: 
This is a Public Preview version, so breaking changes are possible in subsequent releases as we improve the product. To provide feedback, please submit an issue in our [Azure SDK for Java GitHub repo](https://github.com/Azure/azure-sdk-for-java/issues).